02-scalable-processing.py

```diff
@@ -13,9 +13,10 @@
 # COMMAND ----------
 
 # DBTITLE 0,Create a new Database for Test Data
-db = "chedispy_results"
-spark.sql(f"DROP SCHEMA IF EXISTS {db} CASCADE")
-spark.sql(f"CREATE SCHEMA {db}")
+schema = "chedispy_results"
```

Comment:
@dr00b one minor suggestion here, but other than that works great and looks great. Let me know if approved and I'll merge the pull request into main.

Reply:
@zavoraad approved thanks!